packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -68,6 +69,9 @@ class ExpansionTile extends StatefulWidget {
   
   /// A widget to display instead of a rotating arrow icon.
   final Widget trailing;
+  
+  /// Describes if the tile should be initially expanded
```

Comment:
If it only controls the initial state then this should be initiallyExpanded, rather than isExpanded.

The docs should describe what "null" means. If "null" always means "false" then "false" should just be the default value, and null should be disallowed (assert != null in the constructor).

Also, missing trailing period in the dartdoc.